worker-batch-ingestion-container-testing/pom.xml

```diff
@@ -146,7 +146,7 @@
                     <dependency>
                         <groupId>com.github.jobservice</groupId>
                         <artifactId>job-service-contract</artifactId>
-                        <version>9.0.0-SNAPSHOT</version>
+                        <version>9.0.0-530</version>
```

Comment:
Hi @michael-bryson just an fyi I was updating this before I noticed the Job Service was being re-released. I will wait till the re-release is done then update this to the latest 9.0.1-xxx version

Reply:
Ok thats grand, it won't matter too much as the only change was to a db function, not the contract.